pumps/kafka.go

```diff
@@ -54,9 +58,9 @@ func (k *KafkaPump) Init(config interface{}) error {
 	}
 
 	var tlsConfig *tls.Config
-	if k.kafkaConf.UseSSL {
+	if k.kafkaConf.SSLConf.Enabled {
```

Comment:
The pump is not a server. Why do we need to wrap InsecureSkipVerify inside this flag? Maybe we should use this flag if we also load certificates so the `SSLConf.Enabled` role is to turn on/off the mtls, but even for that, I'm not sure it's that necessary.